### PR TITLE
[Feature](bangc-ops): add coverage test doc and revise env descriptions doc

### PR DESCRIPTION
**Pull request by DanieeelLiu**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

        modified:   README.md
        modified:   bangc-ops/README.md
        modified:   docs/Gencase-User-Guide-zh.md
added:
        bangc-ops/env_dump_data_set_helper.sh
        bangc-ops/env_gencase_set_helper.sh
        docs/CoverageTest-User-Guide-zh.md
       
## 2. Modification

 add coverage test doc and revise env descriptions doc

### CODE REVIEW

File: docs/bangc-docs/user_guide/11_env_var/index.rst [hunk 1]
```diff
@@ -6,6 +6,54 @@ Cambricon BANG C OPS库环境变量
 
 本章介绍 Cambricon BANG C OPS 库环境变量。
 
+.. MLUOP_BUILD_GTEST:
```
Comment (AquamarineSu) [inline docs/bangc-docs/user_guide/11_env_var/index.rst:9]: 链接都需要价格下划线哦~ 不然会断链

```suggestion
.. _MLUOP_BUILD_GTEST:
```

File: docs/bangc-docs/user_guide/11_env_var/index.rst [hunk 2]
```diff
@@ -6,6 +6,54 @@ Cambricon BANG C OPS库环境变量
...
+运行前设置，设置gen_case工具功能等级。
+
+**使用方法**
+
+- export MLUOP_GEN_CASE=0：关闭 gen_case 模块功能。
+- export MLUOP_GEN_CASE=1：生成 prototxt，输入输出只保留 shape 等信息。
+- export MLUOP_GEN_CASE=2：生成 prototxt,并保留输入真实真。
+- export MLUOP_GEN_CASE=3：不生成 prototxt,只在屏幕上打印输入输出的 shape 等信息。
+
+更详细请参见 https://github.com/Cambricon/mlu-ops/blob/master/docs/Gencase-User-Guide-zh.md。
```
Comment (AquamarineSu) [inline docs/bangc-docs/user_guide/11_env_var/index.rst:55]:
```suggestion
更详细请参见 `MLU-OPS GEN_CASE 使用指南<https://github.com/Cambricon/mlu-ops/blob/master/docs/Gencase-User-Guide-zh.md>`_ 。
```

File: bangc-ops/README.md [hunk 3]
```diff
@@ -133,3 +133,36 @@ cd bangc-ops/build/test/
 | [core](core)         | 包含公共数据类型的操作、运行时管理、日志等公共实现。           |
 | [kernels](kernels)   | 算子代码实现，包含一元、二元算子模板供其他算子调用。           |
 | [test](test)         | 存放测试算子用的代码。                                         |
+
+## 常用环境变量
+
+简单环境变量可直接执行以下命令：
+
+```bash
+#使能dump data
```
Comment (PetrelYy) [inline bangc-ops/README.md:142]: 注释加个空格
```suggestion
# 使能dump data
```

File: docs/bangc-docs/user_guide/11_env_var/index.rst [hunk 4]
```diff
@@ -6,6 +6,54 @@ Cambricon BANG C OPS库环境变量
...
+
+**功能描述**
+
+运行前设置，设置gen_case工具功能等级。
+
+**使用方法**
+
+- export MLUOP_GEN_CASE=0：关闭 gen_case 模块功能。
+- export MLUOP_GEN_CASE=1：生成 prototxt，输入输出只保留 shape 等信息。
+- export MLUOP_GEN_CASE=2：生成 prototxt,并保留输入真实真。
```
Comment (AquamarineSu) [inline docs/bangc-docs/user_guide/11_env_var/index.rst:52]: 目测逗号写成半角的了~ 后半句是不是打错字了？

File: docs/bangc-docs/user_guide/11_env_var/index.rst [hunk 5]
```diff
@@ -60,26 +108,97 @@ MLUOP_LOG_COLOR_PRINT
...
+
+**功能描述**
+
+作用是在gdram前后刷NAN/INF。
+**使用方法**
+
+- export MLUOP_GTEST_SET_GDRAM=NAN：在gdram前后刷NAN。
+- export MLUOP_GTEST_SET_GDRAM=INF：在gdram前后刷INF。
+
+若不设置，偶数天刷NAN，奇数天刷INF。
```
Comment (AquamarineSu) [inline docs/bangc-docs/user_guide/11_env_var/index.rst:178]: 这句再确认下~

File: docs/bangc-docs/user_guide/11_env_var/index.rst [hunk 6]
```diff
@@ -60,26 +108,97 @@ MLUOP_LOG_COLOR_PRINT
...
+JOB_LIMIT 和CLUSTER_LIMIT 需要同时设置来保证合法性。原理是：1的二进制是0000,0001: 1号cluster可用; 3的二进制是0000,0011: 1号和2好cluster可用; 如果有特殊需求，如只想用2号cluster:设置为2: 0000,0010。
+
+.. MLUOP_GTEST_SET_GDRAM:
+
+MLUOP_GTEST_SET_GDRAM
+#######################
+
+**功能描述**
+
+作用是在gdram前后刷NAN/INF。
```
Comment (AquamarineSu) [inline docs/bangc-docs/user_guide/11_env_var/index.rst:172]: GDRAM是不是统一大小写呀

File: docs/bangc-docs/user_guide/11_env_var/index.rst [hunk 7]
```diff
@@ -60,26 +108,97 @@ MLUOP_LOG_COLOR_PRINT
...
 
-- export MLUOP_GEN_CASE=0 或 unset MLUOP_GEN_CASE：关闭算子调试信息保存功能。
+MLUOP_GTEST_CLUSTER_LIMIT_CAPABILITY
+######################################
 
-默认值为0。
+**功能描述**
+
+设置最大JOB限制数量，默认不设置。
+**使用方法**
```
Comment (AquamarineSu) [inline docs/bangc-docs/user_guide/11_env_var/index.rst:155]: 需要加换行，不然生成格式会很奇怪的
```suggestion

**使用方法**
```

File: docs/CoverageTest-User-Guide-zh.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+### 1. 编译
+
+当前mlu-ops采用了 -c 编译选项 ，指定编译板卡和算子名，编译方式：
+
+```
+$cd mlu-ops
+mlu-ops$cd bangc-ops
+#若NEUWARE_HOME不是默认路径则一定要指定NEUWARE_HOME，否则出现llvm-protobuf: command not found
+./build.sh -c --mlu370 --filter=*abs*
+```
```
Comment (PetrelYy) [inline docs/CoverageTest-User-Guide-zh.md:14]: 11 行有路径  "mlu-ops$"，建议13行保持一致

```suggestion
# 若NEUWARE_HOME不是默认路径则一定要指定NEUWARE_HOME，否则出现llvm-protobuf: command not found
$cd mlu-ops
mlu-ops$ cd bangc-ops
mlu-ops$  ./build.sh -c --mlu370 --filter=*abs*
```
```

File: bangc-ops/README.md (hunk 3 above)
Comment (DanieeelLiu) [reply bangc-ops/README.md]: Fixed

File: docs/bangc-docs/user_guide/11_env_var/index.rst (hunk 1 above)
Comment (DanieeelLiu) [reply docs/bangc-docs/user_guide/11_env_var/index.rst]: Fixed

File: docs/bangc-docs/user_guide/11_env_var/index.rst (hunk 2 above)
Comment (DanieeelLiu) [reply docs/bangc-docs/user_guide/11_env_var/index.rst]: Fixed

File: docs/bangc-docs/user_guide/11_env_var/index.rst (hunk 4 above)
Comment (DanieeelLiu) [reply docs/bangc-docs/user_guide/11_env_var/index.rst]: Fixed

File: docs/bangc-docs/user_guide/11_env_var/index.rst (hunk 6 above)
Comment (DanieeelLiu) [reply docs/bangc-docs/user_guide/11_env_var/index.rst]: fixed

File: docs/bangc-docs/user_guide/11_env_var/index.rst (hunk 7 above)
Comment (DanieeelLiu) [reply docs/bangc-docs/user_guide/11_env_var/index.rst:157]: Fixed

File: docs/CoverageTest-User-Guide-zh.md (hunk 8 above)
Comment (DanieeelLiu) [reply docs/CoverageTest-User-Guide-zh.md]: Fixed

File: docs/bangc-docs/user_guide/11_env_var/index.rst [hunk 9]
```diff
@@ -6,6 +6,54 @@ Cambricon BANG C OPS库环境变量
...
+**功能描述**
+
+运行前设置，设置gen_case工具功能等级。
+
+**使用方法**
+
+- export MLUOP_GEN_CASE=0：关闭 gen_case 模块功能。
+- export MLUOP_GEN_CASE=1：生成 prototxt，输入输出只保留 shape 等信息。
+- export MLUOP_GEN_CASE=2：生成 prototxt,并保留输入真实真。
+- export MLUOP_GEN_CASE=3：不生成 prototxt,只在屏幕上打印输入输出的 shape 等信息。
```
[the remark this answers is not in the document]
Comment (DanieeelLiu) [reply docs/bangc-docs/user_guide/11_env_var/index.rst]: Fixed

File: docs/bangc-docs/user_guide/11_env_var/index.rst (hunk 5 above)
Comment (DanieeelLiu) [reply docs/bangc-docs/user_guide/11_env_var/index.rst]: 确认了

**Review by AquamarineSu — APPROVED** [review]

**Review by PetrelYy — APPROVED** [review]

**Review by mahxn0 — APPROVED** [review]

**Review by duzekunKTH — APPROVED** [review]

**Pull request merged**